src/StudentList.java

```diff
@@ -26,31 +27,26 @@ public static Student getStudentByIdV1(int id) {
         Student student1 = students.stream()
```

Comment:
no need for assignment 
you can use
 return students.stream()
                .filter(student -> student.getId() == id)
                .findFirst()
                 .orElseThrow(()->new RuntimeException("Student Not Found"));

Reply:
Noted.And will be modified